### PR TITLE
Change manager server getServerAddr to configure STUDIO_IP

### PR DESCRIPTION
**Pull request by wangbokun**
# Proposed changes

## Problem Summary:

When the server has multiple IP addresses, the server sends a startup command to the client, and the server address is invalid,Configure the manager IP directly into the manger configure.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: manager/dm-server/src/main/java/org/apache/doris/stack/control/manager/ResourceNodeAndAgentManager.java [hunk 1]
```diff
@@ -465,12 +462,7 @@ private boolean checkEventBeCancelled(long eventId) {
      * get server address
      */
     private String getServerAddr() {
-        String host = null;
-        try {
-            host = InetAddress.getLocalHost().getHostAddress();
-        } catch (UnknownHostException e) {
-            throw new ServerException("get server ip fail");
-        }
+        String host = System.getenv(EnvironmentDefine.STUDIO_IP_ENV);
```
Comment (JNSimba) [inline manager/dm-server/src/main/java/org/apache/doris/stack/control/manager/ResourceNodeAndAgentManager.java:465]: It is recommended that STUDIO_IP is optional, keep the previous logic, and replace when STUDIO_IP is not empty

**Comment by wangbokun** [comment]
choose configure write fixed STUDIO_IP or automatically ip address

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by JNSimba — DISMISSED** [review]
LGTM

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,21 @@ flink-doris-connector/target/
...
+manager/doris-manager.iml
+manager/general/general.iml
+manager/manager-server/manager-server.iml
+manager/manager-server/src/main/resources/web-resource/
+manager/manager-server/target/
+manager/manager/manager.iml
+manager/manager/target/
+manager/resource-common/resource-common.iml
+manager/resource-common/target/
+manager/general/target/
```
Comment (JNSimba) [inline .gitignore:23]: manager/**/target
manager/**/*.iml

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**